packages/apify/src/enqueue_links/click_elements.ts

```diff
@@ -355,18 +362,24 @@ export async function clickElements(page, selector) {
 /* istanbul ignore next */
 /**
  * This is an in browser function!
- * @param {Element} el
- * @param {number} zIndex
  */
-function updateElementCssToEnableMouseClick(el, zIndex) {
+function updateElementCssToEnableMouseClick(el: Element, zIndex: number): void {
```

Comment:
type this as `HTMLElement`, that way there is no need for the ignore comments. also `el.style.top/left` should be numbers (you will see that when you use `HTMLElement`)

```suggestion
function updateElementCssToEnableMouseClick(el: HTMLElement, zIndex: number): void {
```

Reply:
@B4nan did not get this one - TS expects zIndex/left/top to be strings. Found somewhere at SO that they are implicitly converted to strings. Should the error be expected in this case or I should just convert them to string?